src/components/call/TargetInfo.jsx

```diff
@@ -77,11 +78,34 @@ export default class TargetInfo extends React.Component {
                         });
                         return false;
                     };
+                } else if (caller.get('voip_embed_url')) {
+                    const voipURL = caller.get('voip_embed_url') + '#' + new URLSearchParams({
```

Comment:
Could this have used the environment variable directly?

```suggestion
                } else if (process.env.VOIP_EMBED_URL) {
                    const voipURL = process.env.VOIP_EMBED_URL + '#' + new URLSearchParams({
```

Reply:
is process.env.VOIP_EMBED_URL is not available in the front end code? I tried it and process.env is always empty.